src/lamp_py/runtime_utils/env_validation.py

```diff
@@ -45,9 +45,10 @@ def validate_environment(
         value = os.environ.get(key, None)
         if value is None:
             missing_required.append(key)
-        # do not log private variables
-        if key in private_variables:
-            value = "**********"
+        else:
+            # do not log private variables
+            if key in private_variables:
+                value = "**********"
```

Comment:
we were only logging private vars before 👎 . log them all now.

Reply:
I don't think this needs to be in an `else` block

the `metadata[key] = value` line below just needs to be un-indented one level. 

This was introduced here: https://github.com/mbta/lamp/commit/36b959265ff977425013732019088fe46082d827

when the `add_metadata` calls were removed.